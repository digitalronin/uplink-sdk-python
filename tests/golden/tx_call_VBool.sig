r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 13414990120701867317886468391478504614666104665937807430040757901337046663805
