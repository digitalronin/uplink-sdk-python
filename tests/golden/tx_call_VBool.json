{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VBool", 
                        "contents": true
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNzY3ODMzMzIzMzUwODkxMDk2OTM2NjIxMjMzMzI4MjU2MDM4MDY2MzY1ODE4OTM3NjY2MDg3NjgwMjQzODQ2MDc0Njk5OTIxNjg4Ng==", 
    "timestamp": 1231006505
}